scheduler/docs/faq.md

```diff
@@ -26,3 +26,16 @@ can simply query cook with the intended job uuid to see if it exists
 Cook has the concept of "mea-culpa" retries, for instance when a job is killed due to pre-emption. Some users may prefer for jobs to
 just fail in this case instead of attempting to restart it (e.g. when a job is not idempotent.) To prevent this behavior and ensure
 the job is only launched once, set `max-retries: 1` and `disable-mea-culpa-retries: true`.
+
+## Does Cook set any environment variables before running a task?
+
+The following environment variables are set by Cook before running a task:
+- COOK_JOB_UUID: represents the UUID of the Job the task belongs to.
+- COOK_JOB_GROUP_UUID: represents the UUID of the Group the task belongs to.
```

Comment:
I would make "group" lowercase, and maybe change to "job group"?

Reply:
Done.